app/controllers/admin/merchants_controller.rb

```diff
@@ -1,19 +1,24 @@
 class Admin::MerchantsController < Admin::BaseController
-  def index 
+  def index
     @merchants = Merchant.all
   end
 
-  def show 
+  def show
     @merchant = Merchant.find(params[:id])
   end
 
-  def update 
+  def update
     merchant = Merchant.find(params[:id])
     if merchant.enabled?
       merchant.update(enabled?: false)
       merchant.deactivate_items
       flash[:notice] = "#{merchant.name} is now disabled."
       redirect_to admin_merchants_path
+    else
+      merchant.update(enabled?: true)
+      merchant.activate_items
+      flash[:notice] = "#{merchant.name} is now enabled."
+      redirect_to admin_merchants_path
```

Comment:
adds the other "enable" half to existing "disable" function

Reply:
This might be part of the conflicts because I think Jordan had this method in his last PR.

Reply:
merchants_controller.rb conflict
```
  def update
    merchant = Merchant.find(params[:id])
    if merchant.enabled?
      merchant.update(enabled?: false)
      merchant.deactivate_items
      flash[:notice] = "#{merchant.name} is now disabled."
      redirect_to admin_merchants_path
<<<<<<< jg_us52-54_admin_merchant_user_pages
    else
=======
    else  
>>>>>>> master
      merchant.update(enabled?: true)
      merchant.activate_items
      flash[:notice] = "#{merchant.name} is now enabled."
      redirect_to admin_merchants_path
    end
  end
```
the conflict exists inside just one method. it had `else` and i put `else` and github doesn't know which one to use... some of these conflicts are frustrating

Reply:
Fixed this!